docs/source-app/quickstart.rst

```diff
@@ -49,38 +46,39 @@ And that's it!
 
     .. code-block:: console
 
-        INFO: Your app has started. View it in your browser: http://http://127.0.0.1:7501
+        Your Lightning App is starting. This won't take long.
+        INFO: Your app has started. View it in your browser: http://127.0.0.1:7501/view
 
-        INFO: Running train_script: .../lightning/demo/quick_start/train/train.py
         Global seed set to 42
-        GPU available: False, used: False
+
+        GPU available: True (mps), used: False
         TPU available: False, using: 0 TPU cores
         IPU available: False, using: 0 IPUs
         HPU available: False, using: 0 HPUs
 
-        | Name    | Type     | Params
-        -------------------------------------
-        0 | model   | Net      | 1.2 M
-        1 | val_acc | Accuracy | 0
-        -------------------------------------
+          | Name    | Type     | Params | In sizes       | Out sizes
+        ------------------------------------------------------------------
+        0 | model   | Net      | 1.2 M  | [1, 1, 28, 28] | [1, 10]
+        1 | val_acc | Accuracy | 0      | ?              | ?
+        ------------------------------------------------------------------
         1.2 M     Trainable params
         0         Non-trainable params
         1.2 M     Total params
-        4.800     Total estimated model params size (MB)
-        Epoch 3: 100%|█████████████████████████| 8/8 [00:03<00:00,  2.35it/s, loss=1.58, v_num=39]
-
-        INFO: Running serve_script: .../lightning/demo/quick_start/serve/serve.py
-        INFO: INFO:     Started server process [4808]
-        INFO: INFO:     Waiting for application startup.
-        INFO: INFO:     Application startup complete.
-        INFO: INFO:     Uvicorn running on http://0.0.0.0:8888 (Press CTRL+C to quit)
+        Epoch 4: 100%|█████████████████████████| 16/16 [00:00<00:00, 32.31it/s, loss=0.0826, v_num=0]
+        `Trainer.fit` stopped: `max_epochs=5` reached.
+
+        Running on local URL:  http://127.0.0.1:62782/
         ...
 
 
-The build command will launch the app admin panel UI. In your app admin, you can track your app's progress, or click on the **Open App** button to view see your app's UI:
+The app will open your browser and show an interactive demo:
+
+.. figure:: https://pl-flash-data.s3.amazonaws.com/assets_lightning/quick-start/qiuck-start-tensorboard-tab.png
+    :alt: Quick Start UI - Model Training Tab
+    :width: 100 %
```

Comment:
I created the new screenshots myself. Check if that's good.

Reply:
They look great. Nice work.